actions/testmo-create-resources/compose_testmo_resources_file.py

```diff
@@ -0,0 +1,46 @@
+#! /usr/bin/env python
```

Comment:
is there a reason the call to `testmo` can't just be inlined in the "action.yml"?

Reply:
The call is simple, the tricky part is that the Python script is parsing JSON input, containing potentially many fields to be added, and thus looping through them to build and run the Testmo CLI tool. It _could_ be done in Bash, but I would much rather use basically _any_ programming language over dealing with a Bash to manipulate the JSON input.

Reply:
yeah, I had done the resource file generation in `bash` in the first iteration, but the push back was to do it in python, thus the latest updates.

Reply:
okay.